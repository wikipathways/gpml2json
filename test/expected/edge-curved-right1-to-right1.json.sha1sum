bd307db77f50a1d2f4cdbc173da70eab49d9ca83 ?-
