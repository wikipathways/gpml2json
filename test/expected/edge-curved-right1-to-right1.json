{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["dac07","ce66e","f1f5c"],"drawAs":"rect","gpmlElementName":"Pathway","height":137.89519515420886,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":206.04717118835333,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/81e14a7f01d7f31c3071ccf3ef1645a93f4ef134ff81afebbeab05cd529d5b0e"},"entityMap":{"ce66e":{"id":"ce66e","textContent":"GeneProduct","wpType":"GeneProduct","x":96.04717118835333,"y":87.89519515420886,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32975,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"f1f5c":{"id":"f1f5c","textContent":"0009","wpType":"GeneProduct","x":21.86817699255471,"y":55.127248002288866,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32976,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"dac07":{"id":"dac07","drawAs":"CurvedLine","borderWidth":1,"zIndex":32974,"points":[{"isAttachedTo":"f1f5c","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":101.86817699255471,"y":65.12724800228887,"orientation":[1,0]},{"y":81.51122157824886,"x":196.04717118835333},{"isAttachedTo":"ce66e","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":176.04717118835333,"y":97.89519515420886,"orientation":[-1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["f1f5c","ce66e"]}}}
